src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,120 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_PASSWORD = "123456";
+    private static final String DEFAULT_LOGIN = "123456";
+    private static final String SOME_LOGIN = "some login";
+    private static final String USER1_LOGIN = "user1Login";
+    private static final String USER2_LOGIN = "user2Login";
+    private static final String SHORT_LOGIN = "short";
+    private static final Integer DEFAULT_AGE = 25;
+    private static final int UNCORRECT_AGE = 4;
```

Comment:
I consider constants in tests a bad pattern

simply put - it makes your test methods less readable, you always need to scroll to find a corresponding value
apart from that, it makes your test method depend on a single constant, which gets even worse, if you use that constant in several places

while constants are ok for production code, they're not ok for test code :)

I recommend reading this [article](https://medium.com/@erik.sacre/on-tdd-literals-versus-constants-in-tests-8ac992c7873f)